openfisca_us/parameters/benefit/school_meals/income_limit.yaml

```diff
@@ -0,0 +1,15 @@
+description: Income limit as share of poverty line
+
+free:
+  2020-07-01: 1.3
+  2021-07-01: 1.3
```

Comment:
@nikhilwoodruff I forgot what standard we're setting here, should we drop lines when the policy hasn't changed?

Reply:
I think I remember when we talked about it we decided on keeping even redundant lines.